cmd/utils/flags.go

```diff
@@ -225,6 +225,11 @@ var (
 		Usage: `Blockchain garbage collection mode ("full", "archive")`,
 		Value: "full",
 	}
+	TxLookupLimitFlag = cli.Int64Flag{
+		Name:  "txlookuplimit",
+		Usage: "Number of recent blocks to index transactions-by-hash in (default = index all blocks)",
+		Value: 0,
```

Comment:
https://github.com/ethereum/go-ethereum/pull/20270/files#r346566579 :trollface: 

I think you were correct, please apply the diff